src/solvers/refinement/string_constraint.h

```diff
@@ -116,98 +116,46 @@ inline std::string to_string(const string_constraintt &expr)
 }
 
 /// Constraints to encode non containement of strings.
-class string_not_contains_constraintt : public exprt
+/// string_not contains_constraintt are formula of the form:
```

Comment:
:pick: Typo: are formula -> are formulas (or formulae?)
Also typo in commit message: fields that have named -> fields that have names